internal/git/commands_test.go

```diff
@@ -873,332 +1364,6 @@ func TestBackendCommands(t *testing.T) {
 		}
 	})
 
-	t.Run("ParseVerboseBranchesOutput", func(t *testing.T) {
```

Comment:
Since you delete the tests for `ParseVerboseBranchesOutput`, can you also delete that function?

Reply:
I'm going to delete it in another PR that changes the branch parsing. For now, `ParseVerboseBranchesOutput` is still used by `BranchesSnapshot`.